README.md

```diff
@@ -0,0 +1,46 @@
+# paascharmgen
+
+`paascharmgen` is a tool to generate Go structs for a 12 factor Go app generated using rockcraft and charmcraft
+go-framework extensions.
+
+
+## Usage
+
+Create a Kubernetes charm for a Go project following the [tutorial (TODO)](https://juju.is/docs/sdk/write-your-first-kubernetes-charm-for-a-go-app) using
+the rockcraft and charmcraft go-framework.
+
+You can choose to install the tool with `go install github.com/canonical/paascharmgen@latest` or run it directly without install.
+
+You can add a directive similar to the following one to one of your Go files, replacing the placeholders:
+`//go:generate paascharmgen -c [CHARMCRAFT_YAML_DIRECTORY] -o [OUTPUT_GO_FILE] -p [PACKAGE_NAME]`
+
+Without installing, you can create a directive like the following one:
+`//go:generate go run github.com/canonical/paascharmgen@latest -c [CHARMCRAFT_YAML_DIRECTORY] -o [OUTPUT_GO_FILE] -p [PACKAGE_NAME]`
```

Comment:
Re the square brackets used in go instead of `<...>` in go?

Reply:
sure. Updated.